### PR TITLE
Fix for create table with with clause

### PR DESCRIPTION
**Pull request by collerek**
Close: #155

### CODE REVIEW

**Comment by collerek** [comment]
I branched it from resolve_subqueries (#154) so after we merge that one this one should have only relevant changes.

File: test/test_create_table.py
Comment (macbre) [inline test/test_create_table.py:93]: Can we also assert on query type here?

**Comment by macbre** [comment]
Let's wait for #154 to merged before this one - hence converted this PR to draft.

File: test/test_create_table.py
Comment (macbre) [reply test/test_create_table.py:93]: Actually it'd be a good idea to add this assertions in the test cases above as well 🙂

Comment (collerek) [reply test/test_create_table.py:95]: Sure, done

**Comment by tipanverella** [comment]
As a stargazer :wink: , I was wondering when this pull request was going to be scheduled.

File: .github/workflows/python-ci.yml
Comment (macbre) [inline .github/workflows/python-ci.yml:72]: Sorry for that and thanks for fixing.

Comment (collerek) [reply .github/workflows/python-ci.yml:72]: No worries 🙂

File: test/test_create_table.py
Comment (macbre) [inline test/test_create_table.py:103]:
```suggestion
    parser = Parser(qry)
    assert parser.query_type == QueryType.CREATE
```

**Review by macbre — APPROVED** [review]
All good 🙂

File: test/test_create_table.py
Comment (collerek) [reply test/test_create_table.py:103]: Yep it will pass and did just merged

**Comment by macbre** [comment]
Closes #160 too.

**Pull request merged**
